### DISCUSSION TITLE
Dollar Year of Inputs

### DISCUSSION THREAD

**Discussion by jikac** [NREL/REopt-API-Analysis#63] [5 - Financial Input Questions] [answered]
Quick question! My team and I are trying to figure out what the dollar year is of the cost inputs. For example, PV was recently updated to $1592/kW, but we are curious as to whether this figure is in 2022 dollars, or if it gets deflated to the load profile year that is specified with a custom load profile?

Best,

Jesse

**Comment by NLaws** [comment, marked as answer]
All costs are assumed to be in present cost terms. The `LoadProfile.year` input is only for aligning weekends/weekdays with the load and tariff.  "Present cost" is relative to the costs provided by a user (if default costs are not used). For example, if you wish to evaluate decisions with REopt using forecasted costs then "present" is whatever year you choose.
